gitlab4j-models/src/main/java/org/gitlab4j/api/webhook/BuildEvent.java

```diff
@@ -42,6 +43,8 @@ public class BuildEvent extends AbstractEvent {
     private EventProject project;
     private Runner runner;
 
+    private Environment environment;
```

Comment:
Shouldn't this be `EventEnvironment` instead of `Environment`?

At least this would match my observations…